Components/FlightControl/RocketSM.cpp

```diff
@@ -467,6 +467,16 @@ RocketState Ignition::HandleCommand(Command& cm)
             // This is a transition directly to ARM (no checks required)
             nextStateID = RS_ARM;
             break;
+        case RSC_IGINITION_CMD:
+            SOAR_PRINT("This was for testing purposes...\n");
+            break;
+        case RSC_MANUAL_IGNITION_CONFIRMED:
+            TimerTransitions::Inst().ignitionConformation = true;
```

Comment:
Are we not supposed to have any way to cancel the ignition conformation (should be confirmation but probably too hard to change)